src/components/knapper/StyledKnapper.tsx

```diff
@@ -0,0 +1,67 @@
+import styled from 'styled-components/macro';
```

Comment:
Burde denne filen hete `StyledNavigeringsknapper`?  Hvis vi skal ha "andre knapper" senere så kan det kanskje bli en misforståelse?

Reply:
@charliemidtlyng Oi, ja. Har glemt å endre filen til det samme som komponentnavnet. 
Vi kan også prøve å finne et bedre navn. Dette er jo egentlig bare en styling component uten de faktisk knappene brukt inni her. Forslag til et med forståelig navn?

Reply:
Hmmm, `StyledNavigasjon` / `StyledNavigeringsWrapper`, kanskje... Alltid vanskelig med wrapper-komponenter :p

Reply:
Jeg likte `StyledNavigeringsWrapper` selv om det var litt langt, så tenker jeg å gå for denne 😁  Takk for innspill ✨